core/templates/dev/head/domain/collection/CollectionUpdateService.js

```diff
@@ -32,27 +32,34 @@ oppia.constant('COLLECTION_PROPERTY_CATEGORY', 'category');
 oppia.constant('COLLECTION_PROPERTY_OBJECTIVE', 'objective');
 oppia.constant('COLLECTION_PROPERTY_LANGUAGE_CODE', 'language_code');
 oppia.constant('COLLECTION_PROPERTY_TAGS', 'tags');
+oppia.constant('CMD_ADD_COLLECTION_SKILL', 'add_collection_skill');
+oppia.constant('CMD_DELETE_COLLECTION_SKILL', 'delete_collection_skill');
 oppia.constant(
-  'COLLECTION_NODE_PROPERTY_PREREQUISITE_SKILLS', 'prerequisite_skills');
-oppia.constant('COLLECTION_NODE_PROPERTY_ACQUIRED_SKILLS', 'acquired_skills');
+  'COLLECTION_NODE_PROPERTY_PREREQUISITE_SKILLS', 'prerequisite_skill_ids');
```

Comment:
Should these constants also be renamed, per the similar comment earlier in collection_domain?